samples/extensions/sparse_image/sparse_image.h

```diff
@@ -0,0 +1,375 @@
+/* Copyright (c) 2023, Mobica Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 the "License";
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include "api_vulkan_sample.h"
+
+class SparseImage : public ApiVulkanSample
+{
+  public:
+	enum class Stages : uint8_t
```

Comment:
Hmm it fits the required size. Can it cause any problems?

Reply:
No, it should not cause any problems.
I just wanted to know why you explicitly specify the type. Usually, it would only be needed if it's also used in some size-specific (not type-specific) interface.